tests/foreman/ui/test_rhc.py

```diff
@@ -0,0 +1,163 @@
+            organization=module_rhc_org,
+            environment=module_target_sat.api.LifecycleEnvironment(id=module_rhc_org.library.id),
+            auto_attach=True,
+        ).create()
+        default_subscription = module_target_sat.api.Subscription(
+            organization=module_rhc_org
+        ).search(query={'search': f'name="{DEFAULT_SUBSCRIPTION_NAME}"'})[0]
+        ak.add_subscriptions(data={'quantity': 10, 'subscription_id': default_subscription.id})
+        logger.debug(f"Activation key: {ak} \n Organization: {module_rhc_org}")
+
+
+@pytest.mark.tier3
+def test_positive_configure_cloud_connector(
+    session,
+    module_target_sat,
+    fixture_enable_rhc_repos,
+    module_rhc_org,
+    fixture_setup_rhc_satellite,
+):
+    """Install Cloud Connector through WebUI button
+
+    :id: 67e45cfe-31bb-51a8-b88f-27918c68f32e
+
+    :Steps:
+
+        1. Navigate to Configure > Inventory Upload
+        2. Click Configure Cloud Connector
+        3. Open the started job and wait until it is finished
+
+    :expectedresults: The Cloud Connector has been installed and the service is running
+
+    :CaseLevel: Integration
+
+    :CaseImportance: Critical
+
+    :BZ: 1818076
+    """
+    # Copy foreman-proxy user's key to root@localhost user's authorized_keys
+    module_target_sat.add_rex_key(satellite=module_target_sat)
+
+    # Set Host parameter source_display_name to something random.
+    # To avoid 'name has already been taken' error when run multiple times
+    # on a machine with the same hostname.
+    host = module_target_sat.api.Host().search(query={'search': module_target_sat.hostname})[0]
+    parameters = [{'name': 'source_display_name', 'value': gen_string('alpha')}]
+    host.host_parameters_attributes = parameters
+    host.update(['host_parameters_attributes'])
+
+    with session:
+        session.organization.select(org_name=module_rhc_org.name)
+        if session.cloudinventory.is_cloud_connector_configured():
+            pytest.skip(
+                'Cloud Connector has already been configured on this system. '
+                'It is possible to reconfigure it but then the test would not really '
+                'check if everything is correctly configured from scratch. Skipping.'
+            )
+        session.cloudinventory.configure_cloud_connector()
+
+    template_name = 'Configure Cloud Connector'
+    invocation_id = (
+        module_target_sat.api.JobInvocation()
+        .search(query={'search': f'description="{template_name}"'})[0]
+        .id
+    )
+    wait_for(
+        lambda: module_target_sat.api.JobInvocation(id=invocation_id).read().status_label
+        in ["succeeded", "failed"],
+        timeout="1500s",
+    )
+
+    job_output = module_target_sat.cli.JobInvocation.get_output(
+        {'id': invocation_id, 'host': module_target_sat.hostname}
+    )
+    logger.debug(f"Invocation output>>\n{job_output}\n<<End of invocation output")
```

Comment:
non-blocking
I believe that also this can go away as you use CLI JobInvocation which is executed by broker. Broker logs stdout from hammer to robottelo.log.

Reply:
removed :)